### DISCUSSION TITLE
Event listing

### DISCUSSION THREAD

**Discussion by rms59** [nguyenquyhy/Flight-Tracker-StreamDeck#332] [General]
I would like to alphabetically sort the KnownEvents.cs listing for my personal use but every .cs source code editor I find won't open it. Any suggestions?

**Comment by nguyenquyhy** [comment]
Maybe you can use the sim documentation directly. https://docs.flightsimulator.com/html/index.htm#t=Programming_Tools%2FSimVars%2FEvent_IDs.htm

If you want to read the .cs file, any text editor would do (e.g. Notepad, Word, etc.)

**Comment by rms59** [comment]
As aways, thanks. I didn't see the indexed one but that looks like what I want. Still learning how to navigate GitHub :) 
Thanks again!
